package.json

```diff
@@ -367,30 +372,35 @@
           "group": "navigation@3"
         },
         {
-          "command": "dbtPowerUser.testCurrentModel",
+          "command": "dbtPowerUser.sqlToModel",
```

Comment:
move last as this is probably least used navigation item.

Reply:
moved to last.